README.md

```diff
@@ -40,17 +40,17 @@ Designing in the Browser is a methodology that directly uses the browser as the
 Benefits
 --------
 
-- Requires minimal skills to get started--just HTML
+- Requires minimal skills to get started (Just HTML)
 - Creates accessible prototypes
-- Once learned, much faster than any drag-and-drop tool
-- Create dynamic designs that flex and change depending on the width of the screen
+- Once learned,then it provides Experience of much faster than any drag-and-drop tool.
```

Comment:
This is grammatically incorrect, but I see what you are trying to say. How about 'Once learned, it provides a much faster prototyping experience than visual drag-and-drop tools.'